lib/websocket.py

```diff
@@ -34,6 +34,8 @@ class SSLError(Exception):
     HAVE_SSL = False
 
 from six.moves.urllib.parse import urlparse
+from six.moves import range
+from six import ensure_binary, ensure_text, iteritems, PY3
```

Comment:
Why don't we use `six.PY3` in `lib/utils.py` as well? There we define our own `PY3`. And I'll have to check but I'm pretty sure I could use `ensure_binary()` and `ensure_tex()` in other places as well.

Reply:
Yes, since the addon depends on six I'd also use it in utils. I'll add a commit for it.

Reply:
Done in https://github.com/Montellese/mediaimporter.plex/pull/4/commits/bc12f75ecf81f318518c8dfdf7c944806a1d6e3c

Reply:
Thanks